src/Telemetry/AzureDevOpsTimeline/NullKustoIngestClient.cs

```diff
@@ -0,0 +1,47 @@
+using Kusto.Ingest;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline
+{
+    public class NullKustoIngestClient : IKustoIngestClient
+    {
+        private static IKustoIngestionResult SuccessfulResultInstance { get; } = new AlwaysSuccessIngestionResult();
```

Comment:
I'd also make it a field.  There's not really any reason for it to be a property.